src/main/java/dev/jbang/cli/Export.java

```diff
@@ -378,3 +385,85 @@ private Path getFatjarOutputPath() {
 		return outputPath;
 	}
 }
+
+@Command(name = "jlink", description = "Exports a minimized JDK distribution")
+class ExportJlink extends BaseExportCommand {
+
+	@CommandLine.Parameters(index = "1..*", arity = "0..*", description = "Parameters to pass on to the jlink command")
+	public List<String> params = new ArrayList<>();
```

Comment:
we normally  treat arguments that comes after the script as args to the script not to the underlying tools.

should this not be a dedicated arg instead?

Reply:
In this case it's the intention that _all_ extra parameters are passed directly on to the underlying `jlink` command. This way you have the full feature set of that command without us having to add all those options.

Reply:
Btw, the treating parameters after the script as arguments to the script is only true the `run` and `alias` commands (and `alias` is basically a delayed `run` command), so IMO we're free to re-interpret that for other commands that are not run-related.

Reply:
we somewhat did it for jbang edit too - but with the new edit that won't hold up.

still - this thing of passing directly to jlink feels wrong to me.... can you give an example of how/what it would look like? what arguments would you want to put there?

Reply:
Well this is what jlink supports:

```
      --add-modules <mod>[,<mod>...]    Root modules to resolve
      --bind-services                   Link in service provider modules and
                                        their dependences
  -c, --compress=<0|1|2>                Enable compression of resources:
                                          Level 0: No compression
                                          Level 1: Constant string sharing
                                          Level 2: ZIP
      --disable-plugin <pluginname>     Disable the plugin mentioned
      --endian <little|big>             Byte order of generated jimage
                                        (default:native)
  -h, --help, -?                        Print this help message
      --ignore-signing-information      Suppress a fatal error when signed
                                        modular JARs are linked in the image.
                                        The signature related files of the
                                        signed modular JARs are not copied to
                                        the runtime image.
      --launcher <name>=<module>[/<mainclass>]
                                        Add a launcher command of the given
                                        name for the module and the main class
                                        if specified
      --limit-modules <mod>[,<mod>...]  Limit the universe of observable
                                        modules
      --list-plugins                    List available plugins
  -p, --module-path <path>              Module path
      --no-header-files                 Exclude include header files
      --no-man-pages                    Exclude man pages
      --output <path>                   Location of output path
      --post-process-path <imagefile>   Post process an existing image
      --resources-last-sorter <name>    The last plugin allowed to sort
                                        resources
      --save-opts <filename>            Save jlink options in the given file
  -G, --strip-debug                     Strip debug information
      --suggest-providers [<name>,...]  Suggest providers that implement the
                                        given service types from the module path
  -v, --verbose                         Enable verbose tracing
      --version                         Version information
      @<filename>                       Read options from file
```

Almost any of those options could be useful to a user and jlink even has some options that are not in this list, you can add `-J` for example to pass options to the underlying JVM for example. And the command can be extended with modules (exactly what I'm using it for) that add their own options to the command dynamically.

So we can't really predict what flags the user will need. Which is why passing on anything seems like the most useful option.

Why does passing those options feel bad to you?

Reply:
mkay.

I'll accept it. 

reason i dont like it is that the command line args for the app is useful...but I realize we could if/when that becomes relevant for some commands we can do like git does and uses -- to separate between branch/flags and additional arguments/files.